Classes/Controller/BackendController.php

```diff
@@ -166,9 +166,9 @@ public function syntaxAction(array $syntax, array $extensions, array $formats, a
 				}
 				$reportsForSyntaxName = array();
 				if ('php' === $syntaxName) {
-					$reportsForSyntaxName = $this->syntaxService->syntaxCheckPhpFilesInPath($extensionFolder . '/Classes', $filteredFiles);
+					$reportsForSyntaxName = $this->syntaxService->syntaxCheckPhpFilesInPath($extensionFolder . '/Classes');
 				} elseif ('fluid' === $syntaxName) {
-					$reportsForSyntaxName = $this->syntaxService->syntaxCheckFluidTemplateFilesInPath($extensionFolder . '/Resources', $csvFormats, $filteredFiles);
+					$reportsForSyntaxName = $this->syntaxService->syntaxCheckFluidTemplateFilesInPath($extensionFolder . '/Resources', $csvFormats);
```

Comment:
The filteredFiles argument should definitely still be supported. It is designed to enable users to select a sub-set of listed files using checkboxes, to only process those files. This function has worked in BE before.